test/musica/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ target_sources(test_musica_api
     ${CCPP_SRC_PATH}/ccpp_hash_table.F90
     ${CCPP_SRC_PATH}/ccpp_hashable.F90
     ${CCPP_SRC_PATH}/ccpp_types.F90
+    ${TO_BE_CCPPIZED_SRC_PATH}/ccpp_const_utils.F90
```

Comment:
I might suggest adding it after line 31, just because they come from the same sources, but it's not a big deal.

Reply:
moved!